concourse/pipelines/bosh-cli/bosh-cli.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+
+image: docker:///governmentpaas/bosh-cli
```

Comment:
The `bosh` commands generate a lot of warnings about OpenSSL. It looks like these come from [`httpclient` because of the OpenSSL` version](https://github.com/nahi/httpclient/blob/85b3fcde9a0378e9a219765b0b79b465162b173f/lib/httpclient/ssl_config.rb#L441-L452) in the [`governmentpaas/bosh-cli` container](https://github.com/alphagov/paas-docker-cloudfoundry-tools/blob/cc3240fad4ed606bb1b5173e32f05db25e32b3c4/bosh-cli/Dockerfile):
```
bosh deployment ./cf-manifest-with-uuid.yml

RSA 1024 bit CA certificates are loaded due to old openssl compatibility
Logged in as `admin'
RSA 1024 bit CA certificates are loaded due to old openssl compatibility
Target set to `my-bosh'
RSA 1024 bit CA certificates are loaded due to old openssl compatibility
RSA 1024 bit CA certificates are loaded due to old openssl compatibility
Deployment set to `/tmp/build/beda48a9-3deb-40e6-54a1-ed5cd322f94e/cf-manifest-with-uuid.yml'
succeeded
```

Is that reason enough to move to the smaller Alpine based image which has a later version of OpenSSL?
```
➜  paas-cf git:(bosh_cli_task) ✗ docker run --rm -ti ruby:2.2-alpine openssl version
OpenSSL 1.0.2f  28 Jan 2016
```